py-polars/polars/utils/udfs.py

```diff
@@ -536,12 +536,7 @@ def _matches(
         n_required_ops, argvals = len(opnames), argvals or []
         instructions = self._instructions[idx : idx + n_required_ops]
         if len(instructions) == n_required_ops and all(
-            any(
-                inst.opname == match_opname
-                if match_opname[-1] != "*"
-                else inst.opname.startswith(match_opname[:-1])
-                for match_opname in match_opnames
-            )
+            any(inst.opname == match_opname for match_opname in match_opnames)
```

Comment:
Isn't this just `inst.opname in match_opnames` now?

Reply:
thanks!